### DISCUSSION TITLE
QoL in RSL Helper

### DISCUSSION THREAD

**Discussion by Moizevitch** [FarbstoffRSL/RSL-Helper#523] [Ideas]
Hello Mr Farbstoff! If possible, I would like to suggest an evolution tip (or QoL) for RSL Helper: in the classic arena (and in 3 vs 3 too) there are 10 opposing players against whom we can fight. My idea is simple: in the RSL Helper tool, there would be an area with 10 squares representing each of the 10 players. Then, the user would choose which opponents (for example 2, 4, 5 8, and 10) he/she would pick to fight with. Then, once play is given, the tool would carry out the fights by clicking where you need to click. In the same way as in the dungeon, for example. If it's not cheating in the dungeon, or the hydra, or iron twins, why would it be in the arenas? I'm farming the arena medals for the Great Hall, so it would be wonderful if I could do several fights in a row, it would save a lot of time too. Please, I don't know how complex it would be to do this, but it is already clear of your great competence and skill. So please (🙏) try to introduce this feature in your tool, it would be something that all users would be very grateful for. Well, that's it. Thanks.

**Comment by Dwza** [comment]
i dont think that this would be allowed by plarium

**Reply by Moizevitch** [reply]
Well, as I said, there is no invasion in the game, just clicks automation. Something that already exists in the tool and Plarium allows it. But, I'll be rooting for this.

**Comment by addiemassey** [comment]
Quality-of-life improvements in [RSL Helper](https://rslhelper.com/) always make grinding easier. I'm looking forward to seeing what’s new in this update.
